smpe/smpe_workflow.xml

```diff
@@ -0,0 +1,1785 @@
+		<variableValue name="mgmtclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="tzone" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="csihlq" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="dzone" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="srel" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="hlq" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="thlq" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="dvol" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="dsclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="ddclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="dmclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="dhlq" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="tvol" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="tsclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="tdclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="tmclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="fsdsn" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="dsprefix" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="fsvol" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="fsclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="fdclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="fmclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="pathPrefix" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="dvol2" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="tvol2" noPromptIfSet="false" required="false" scope="instance"/>
+		<instructions substitution="false">Run this step to initialize variable values</instructions>
+		<weight>1</weight>
+		<autoEnable>false</autoEnable>
+		<canMarkAsFailed>false</canMarkAsFailed>
+	</step>
+	<step name="ZWE1SMPE" optional="false">
+		<title>CSI Allocation</title>
+		<description>Allocates a CSI and data sets required by SMP/E</description>
+		<prereqStep name="define_variables"/>
+		<instructions substitution="false">Allocates a CSI and data sets required by SMP/E</instructions>
+		<weight>1</weight>
+		<autoEnable>true</autoEnable>
+		<canMarkAsFailed>false</canMarkAsFailed>
+		<template>
+			<inlineTemplate substitution="true">#if( $jobCard and $jobCard != &quot;&quot; and $jobCard != '#jobCard')

```

Comment:
inline templates make it hard for maintaining the members. Can we store them externally in Github and merge in here for distribution? Will have to look at zowe build scripts updating the members (like current year and FID, but in the future also ALLOC sizes).

Reply:
Maybe we could, we chose this approach because editing the workflow is easier when it is just one file just like is debugging it.